client/components/gravatar/test/index.jsx

```diff
@@ -22,53 +21,69 @@ function stripReactAttributes( string ) {
 }
 
 describe( 'Gravatar', function() {
-	var bobTester = {
+	const bobTester = {
 		avatar_URL: 'https://0.gravatar.com/avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&d=identicon',
 		display_name: 'Bob The Tester'
 	};
 
 	describe( 'rendering', function() {
 		it( 'should render an image given a user with valid avatar_URL, with default width and height 32', function() {
-			var gravatar = <Gravatar user={ bobTester } />,
-				expectedResultString = '<img alt="Bob The Tester" class="gravatar" src="https://0.gravatar.com/avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&amp;d=mm" width="32" height="32"/>';
+			const gravatar = <Gravatar user={ bobTester } />,
+				expectedResultString = '<img alt="Bob The Tester" class="gravatar" ' +
+				'src="https://0.gravatar.com/' +
+				'avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&amp;d=mm" ' +
+				'width="32" height="32"/>';
 
 			assert.equal( expectedResultString, stripReactAttributes( ReactDomServer.renderToStaticMarkup( gravatar ) ) );
```

Comment:
@dmsnell - these tests were written before my time. I'm not sure that updating the tests for this PR falls under the scope here :) The tests I wrote for a different PR use enzyme <3